test/startup-script/README.md

```diff
@@ -0,0 +1 @@
+Based on https://github.com/kubernetes-retired/contrib/tree/master/startup-script
```

Comment:
Maybe explain here how/when to rebuild this image? Or do we already have something else ?

Reply:
We have an automated build for it now, after this PR is merged I will make a follow up PR for changing the image tag to master.